fiware-region-sanity-tests/docker/Dockerfile

```diff
@@ -9,28 +13,30 @@ RUN wget --no-check-certificate http://pypi.python.org/packages/source/d/distrib
 RUN tar xf distribute-0.6.35.tar.gz
 RUN cd distribute-0.6.35;python2.7 setup.py install
 RUN easy_install-2.7 pip
-RUN pip2.7 install virtualenv
-RUN virtualenv /root/venv/fiware-region-sanity-tests --system-site-packages
-RUN echo "source /root/venv/fiware-region-sanity-tests/bin/activate" >>/etc/bashrc
+RUN yum install -y dbus-devel.x86_64 dbus-glib-devel.x86_64
```

Comment:
If this docker execute the tests why do you install dbus? What is the different with the previous one.